src/BuildScriptGenerator/Php/PhpBashBuildSnippet.sh.tpl

```diff
@@ -6,7 +6,7 @@ declare -r composer='/opt/php-composer/composer.phar'
 echo "Composer archive: $composer"
 echo "Running composer install..."
 echo
-php $composer install
+php $composer install --ignore-platform-reqs --no-interaction
```

Comment:
Add a comment as to why we are doing this here

Reply:
Also it would be nice to print out that we are running this command, like we do for npm and yarn install...can give end user an idea of what we are doing...